nipype/info.py

```diff
@@ -133,16 +132,38 @@ def get_nipype_gitversion():
 ISRELEASE = _version_extra == ''
 VERSION = __version__
 PROVIDES = ['nipype']
-REQUIRES = ["nibabel>=%s" % NIBABEL_MIN_VERSION,
-            "networkx>=%s" % NETWORKX_MIN_VERSION,
-            "numpy>=%s" % NUMPY_MIN_VERSION,
-            "python-dateutil>=%s" % DATEUTIL_MIN_VERSION,
-            "scipy>=%s" % SCIPY_MIN_VERSION,
-            "traits>=%s" % TRAITS_MIN_VERSION,
-            "nose>=%s" % NOSE_MIN_VERSION,
-            "future>=%s" % FUTURE_MIN_VERSION,
-            "simplejson>=%s" % SIMPLEJSON_MIN_VERSION,
-            "prov>=%s" % PROV_MIN_VERSION,
-            "mock",
-            "xvfbwrapper"]
+REQUIRES = [
+    "nibabel>=%s" % NIBABEL_MIN_VERSION,
+    "networkx>=%s" % NETWORKX_MIN_VERSION,
+    "numpy>=%s" % NUMPY_MIN_VERSION,
+    "python-dateutil>=%s" % DATEUTIL_MIN_VERSION,
+    "scipy>=%s" % SCIPY_MIN_VERSION,
+    "traits>=%s" % TRAITS_MIN_VERSION,
+    "future>=%s" % FUTURE_MIN_VERSION,
+    "simplejson>=%s" % SIMPLEJSON_MIN_VERSION,
+    "prov>=%s" % PROV_MIN_VERSION,
+    "xvfbwrapper",
+    "funcsigs"
+]
+
+TESTS_REQUIRES = [
+    "nose>=%s" % NOSE_MIN_VERSION,
+    "mock",
+    "codecov",
+    "doctest-ignore-unicode"
+]
+
+EXTRA_REQUIRES = {
+    'doc': ['Sphinx>=0.3'],
+    'tests': TESTS_REQUIRES,
+    'fmri': ['nitime', 'nilearn'],
```

Comment:
the fmri extra or the tests extras? I guess so, this is the best moment to decide the dependencies and how we organize them